gourmet/Undo.py

```diff
@@ -80,7 +80,7 @@ class UndoableTextChange (UndoableObject):
     def __init__ (self, set_text_action, history, initial_text="",text="",txt_id=None,is_undo=False):
         self.txt_id = txt_id
         self.blob_matcher = re.compile('\s+\S+\s+')
-        self.initial_text = initial_text
+        self.initial_text = initial_text if initial_text is not None else ""
```

Comment:
Doesn't `initial_text if initial_text else ""` produce the same result?

Reply:
In that context, yes. However, I find that this formulation is more explicit: there's someone somewhere handing us `None`, and we're explicitly avoiding that value here, as it ruins the `undo` functionality later.